src/python/Utils/HeartbeatMonitorBase.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import (division, print_function)
+from Utils.CherryPyPeriodicTask import CherryPyPeriodicTask
+
+class HeartbeatMonitorBase(CherryPyPeriodicTask):
+    
+    def __init__(self, rest, config):
+        super(HeartbeatMonitorBase, self).__init__(config, enableLogDB=True)
+        
+        from WMCore.Services.WMStats.WMStatsWriter import WMStatsWriter
```

Comment:
The reason I put this in init method is I am not sure which application has dependence on Utils package. 
Those applications won't be necessary using this class. Or I could move this package some where else (under Services) but I thought this might be the better place.